pysph/examples/rb_rotation_matrices/stack_of_cylinders.py

```diff
@@ -0,0 +1,251 @@
+"""
```

Comment:
The first line should be short and should include the run time.  See any of the examples for examples.